src/docs/output-targets/custom-elements.md

```diff
@@ -192,6 +221,8 @@ your package can find the type definitions, like so:
 }
 ```
 
+> Note: If you set the `dir` property on the output target config, replace `dist/components` in the above snippet with the path set in the config.
+
 ## Example Bundler Configs
```

Comment:
Left this alone. I'm not too sure of it's helpfulness, but it looks like it was added ~2 years ago by a community member. Long term, I'd even consider use separating these output target docs into two pages: one for setting up/configuring, and one for consuming the output in a project.